run.sh

```diff
@@ -41,7 +42,5 @@ exportfs -ra
 service nfs-kernel-server start
 
 echo "- Nfs server is up and running.."
-# configure tftp
-sed -i 's/TFTP_DIRECTORY=\"\/var\/lib\/tftpboot\"/TFTP_DIRECTORY=\"\/srv\/tftp\"/' /etc/default/tftpd-hpa
-sed -i 's/secure/secure -c/' /etc/default/tftpd-hpa
-/etc/init.d/tftpd-hpa start
+cd $HTTP_SRV
+python -m SimpleHTTPServer 8000
```

Comment:
`python -m http.server 8000` for python3

Reply:
@mek-x python3 is no default in Debian stable. We would have to change container, does it make sense ? Maybe w should use Go ?